test/tools/verify/cmd/main.go

```diff
@@ -3,37 +3,67 @@ package main
 import (
 	"flag"
 	"fmt"
+	machineclientset "github.com/kubermatic/machine-controller/pkg/client/clientset/versioned"
 	"io/ioutil"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
 	"os"
 	"strings"
 )
 
 func main() {
 	var manifestPath string
 	var parameters string
+	var kubeConfig string
 
+	flag.StringVar(&kubeConfig, "kubeconfig", "", "a path to the kubeconfig.")
 	flag.StringVar(&manifestPath, "input", "", "a path to the machine's manifest.")
-	flag.StringVar(&parameters, "parameters", "", "a list of comma-delimited key value pairs i.e key=value,key1=value2")
+	flag.StringVar(&parameters, "parameters", "", "a list of comma-delimited key value pairs i.e key=value,key1=value2.")
 	flag.Parse()
 
-	if len(manifestPath) == 0 || len(parameters) == 0 {
-		fmt.Println("please specify input and parameters flags")
+	// input sanitizaiton
+	if len(manifestPath) == 0 || len(parameters) == 0 || len(kubeConfig) == 0 {
```

Comment:
we could also let it go and read the `in-cluster` config when the `kubeconfig` was not provided.